ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -677,10 +680,12 @@ public void findEntriesSinceOnRestartTest() throws IOException, StoreException {
     FileSpan secondRecordFileSpan = log.getFileSpanForMessage(firstRecordFileSpan.getEndOffset(), PUT_RECORD_SIZE);
 
     // if there is no bad shutdown but the store token is past the index end offset, it is an error state
-    StoreFindToken startToken = new StoreFindToken(secondRecordFileSpan.getStartOffset(), new UUID(1, 1));
+    StoreFindToken startToken =
+        new StoreFindToken(secondRecordFileSpan.getStartOffset(), new UUID(1, 1), incarnationId, false);
     doFindEntriesSinceFailureTest(startToken, StoreErrorCodes.Unknown_Error);
 
     UUID oldSessionId = sessionId;
+    UUID oldIncarnationId = incarnationId;
```

Comment:
this statement looks out of place. Could you put it before the actual test for incarnation id?